stubs/3.2/queue.pyi

```diff
@@ -6,5 +6,6 @@ from typing import Any
 
 class Queue():
     def get(self, block: bool = True, timeout: float = None) -> Any: pass
+    def put_nowait(self, item: Any) -> None: pass
```

Comment:
You can't find the earlier comment because I tried silently removing it before anyone would notice my mistake :smile:.